showcases/showcase-styles.css

```
@@ -8,7 +8,14 @@
 	display: flex;
 	flex-direction: column;
 	gap: var(--db-spacing-fixed-md);
-	padding: var(--db-spacing-fixed-md);
+
+	&:has(.component-header) {
+		padding: var(--db-spacing-fixed-md);
+	}
+}
+
+.show-code-link {
```

```suggestion
/* TODO: we're disabling the code link temporarily and need to reintegrate it afterwards */
.show-code-link {
```

@michaelmkraus We're only disabling this link temporarily, aren't we? What are the next to this aspect than, as the users might rely on that functionality?